manifests/init.pp

```diff
@@ -374,6 +378,7 @@
   $gitlab_rails = undef,
   $high_availability = undef,
   $logging = undef,
+  $letsencrypt = undef,
```

Comment:
please add a datatype for the new parameter. We try to use them for all new params.

Reply:
good point. Will do.